File: Java/Lista 3/L3_Ex1.4/src/UsuarioRoot.java

public class UsuarioRoot extends Usuario {
    
    public UsuarioRoot(String login) {
        super(login);
    }
    public void descricaoRoot(){
        System.out.println("O usuario Root tem todas as permissões");
    }
    public void excluirTodos(){
        System.out.println("O usuario Root pode excluir qualquer usuario");
    }
    public void despedidaRoot(){
        System.out.println("Até logo" + this.getLogin() + "!");
    }
    
}

File: Swift/L3/Lista_03_EX04/UsuarioRoot.swift
import Foundation
 
 public class UsuarioRoot : Usuario{
 
	func descricaoRoot(){
		print("O usuario Root tem todas as permissões")
	}
	
	func excluirTodos(){
		print("O usuario Root pode excluir qualquer usuario")
	}
	
	func despedidaRoot( ) {
		print("Até logo \(login)!")
	}
	
}